docs/application/dotnet/samples/watch/overview.md

```diff
@@ -1,6 +1,28 @@
 # Watch
```

Comment:
I don't think so. Your suggestion spoils coherence.
However, I changed the code to open the wearable tab when the page is opened. It will improve usability.